JM/CF/Scripts/3_Game/CommunityFramework/TypeConverter/CF_TypeConverter.c

```diff
@@ -0,0 +1,172 @@
+/**
+ * Super implementation to handle all possible conversions. 
+ */
+class CF_TypeConverter : Managed
+{
+	void SetInt(int value)
+	{
+		
+	}
+
+	int GetInt()
+	{
+		return 0;		
+	}
+
+	void SetBool(bool value)
+	{
+		
+	}
+
+	bool GetBool()
+	{
+		return false;
+	}
+
+	void SetFloat(float value)
+	{
+		
+	}
+
+	float GetFloat()
+	{
+		return 0;
+	}
+
+	void SetVector(vector value)
+	{
+		
+	}
+
+	vector GetVector()
+	{
+		return "0 0 0";
+	}
+	
+	void SetString(string value)
+	{
+
+	}
+
+	string GetString()
+	{
+		return "";
+	}
+
+	void SetClass(Class value)
+	{
+		
+	}
+
+	Class GetClass()
+	{
+		return null;
+	}
+
+	void SetManaged(Managed value)
+	{
+		
+	}
+
+	Managed GetManaged()
+	{
+		return null;
+	}
+
+	string ToStr()
+	{
+		return "" + this;
+	}
+
+	void ToIO(CF_IO io)
+	{
+
+	}
+
+	void FromIO(CF_IO io)
+	{
+		
+	}
+
+	void FromVariable(Class instance, string variable)
+	{
+		#ifdef CF_TRACE_ENABLED
+		CF_Trace trace(this, "FromVariable", "" + instance, variable);
+		#endif
+		CF_Log.Error("Override FromVariable!");
+	}
+
+	bool FromTypename(Class instance, int index)
+	{
+		#ifdef CF_TRACE_ENABLED
+		CF_Trace trace(this, "FromVariable", "" + instance, "" + index);
+		#endif
+		CF_Log.Error("Override FromVariable!");
+		return false;
+	}
+
+	void ToVariable(Class instance, string variable)
```

Comment:
I'd call this something like `Apply`/`ApplyTo` because it is a bit more explanatory of what's happening ...

```cs
conveter.ApplyTo(inst, "var");
Print(var); //yay
```

Reply:
It is named as it is to keep consistent with `FromVariable`

Reply:
You read From(a)Variable, which makes sense. But it only makes sense to Apply(To/On) or Write another variable.

Maybe Read() and Write() are better terms for this anyway. ToVariable sounds like you create a new variable instance from something like you do with ToList, ToString, ToWhatever.

Reply:
I'll rename to `Read` and `Write`